nrm_django/nrm_site/templates/admin/base.html

```diff
@@ -6,7 +6,7 @@
 <head>
     <title>{% block title %}{% endblock %}</title>
     {# We've pulled in our own copy of the django admin css so we can cut out parts of it we don't need due to USWDS #}
-    <link rel="stylesheet" type="text/css" href="{% static 'css/admin_base.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/admin_base.css' %}?v=2">
```

Comment:
This is the second time we've had cache-busting creeping into our commits. Shall we log an issue (and feel free to ignore it) to do some SHA-based cache-busting exactly like this?

Reply:
I have no opinion. My feeling is if you need to bust the cache, do so, but I really don't have any opinion on it beyond that.